test/unit/spec/appWidgets/input/select2ObjectinputSpec.js

```diff
@@ -96,13 +96,13 @@ define([
 
     jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
     describe('Select 2 Object Input tests', () => {
-        let bus, testConfig;
+        let bus, testConfig, container;
```

Comment:
`this.node` => `container`
(the reference `this.node` doesn't exist in `afterEach`, so the node can't be removed from the DOM)